src/gretel_trainer/relational/strategies/common.py

```diff
@@ -9,12 +9,14 @@
 from sklearn import preprocessing
 
 from gretel_trainer.relational.core import RelationalData
-from gretel_trainer.relational.sdk_extras import download_file_artifact
+from gretel_trainer.relational.sdk_extras import ExtendedGretelSDK
 
 logger = logging.getLogger(__name__)
 
 
-def download_artifacts(model: Model, out_filepath: Path, table_name: str) -> None:
+def download_artifacts(
```

Comment:
What is `table_name` used for in this function? I didn't see any refs

Reply:
Nice catch. I think it had been used previously, but a refactor made it obsolete and I missed the signature change. The callers of this function use `table_name` to compose `out_filepath`, and no longer need to pass the string down here. Fixing!